modules/integration/src/test/scala/scala/cli/integration/TestTestDefinitions.scala

```diff
@@ -103,6 +103,25 @@ abstract class TestTestDefinitions(val scalaVersionOpt: Option[String])
     )
   )
 
+  val successfulScalaCheckFromCatsNativeInputs = TestInputs(
+    Seq(
+      os.rel / "MyTests.scala" ->
+        """//> using scala "2.13.8"
+          |//> using platform "native"
+          |//> using lib "org.typelevel::cats-kernel-laws::2.8.0"
```

Comment:
```suggestion
          |//> using lib "org.scalacheck::scalacheck::1.16.0"
```

Reply:
But If I replace `cats-kernel-laws` dependency to `scalachek` it works in the actual version of Scala CLI. Therefore I added a test with `cats-kernel-laws` dependency which doesn't work in `0.1.10`

Reply:
Thanks, I didn't realize it was necessary to reproduce the issue.